aten/src/ATen/cpu/vec/vec256/vec256_complex_double.h

```
@@ -318,33 +297,9 @@ template <> class Vectorized<c10::complex<double>> {
   Vectorized<c10::complex<double>> operator!=(const Vectorized<c10::complex<double>>& other) const {
     return _mm256_cmp_pd(values, other.values, _CMP_NEQ_UQ);
   }
-  Vectorized<c10::complex<double>> operator<(const Vectorized<c10::complex<double>>&) const {
```

I wonder what would happen if user calls such operators on complex vec data? On nightly it should raise an error and when we remove it what would be the behaviour?

---

Oh, good point! 

@khushi-411, please only remove dead-code for ATen ops, as we rely upon [Dispatcher to raise errors for unsupported dtypes (https://github.com/pytorch/pytorch/blob/main/aten/src/ATen/Dispatch.h#L222-L228).

---

Makes sense. Thanks, @vfdev-5 and @sanchitintel!

Updated in https://github.com/pytorch/pytorch/pull/110280/commits/96987ac08a360bcadd97a40ec19ac23408e59804